gh-pages/webdx-cg.md

```diff
@@ -0,0 +1,17 @@
+---
+permalink: webdx-cg
+---
+
+# The WebDX Community Group
+
+> The mission of the [Web Developer Experience (WebDX) Community Group](https://www.w3.org/community/webdx/) is to facilitate coordinated approaches to improve the overall experience of developing for the Web platform when such coordination provides unique opportunities for these improvements. — [WebDX Community Group charter](https://github.com/web-platform-dx/admin/blob/main/charter.md).
+
+A subset of the members of the CG make up the owner and governance body for Baseline. The official definition of what Baseline is, including the goals and non-goals of the project, can be found in the [web-platform-dx/web-features repository](https://github.com/web-platform-dx/web-features/blob/main/docs/baseline.md).
+
+The community group also owns the [web feature grouping work](web-features).
```

Comment:
The Venn diagram with WebDX is kinda complicated. I suggest mostly evading this with something like:

```suggestion
The web-features project grew out of the work of the WebDX Community Group. WebDX plays an important part in coordinating the web-features project by hosting meetings and making connections to research, standardization work, and other efforts.
```